web/src/locales/fr.json

```diff
@@ -122,9 +122,8 @@
   "estimation-card": {
     "link": "En savoir plus sur nos modèles d'estimation",
     "estimated_time_slicer_average": {
-      "title": "Les données sont momentanément estimées",
-      "pill": "Indisponibles",
-      "body": "Le fournisseur de données ne publie momentanément aucune donnée pour cette zone. Les valeurs affichées sont estimées en fonction des données historiques et seront remplacées par les données consolidées dès qu'elles seront disponibles. Veuillez noter que les valeurs estimées peuvent être inexactes car elles ne tiennent pas compte d'événements météorologiques spécifiques en temps réel."
+      "title": "Les données sont préliminaires",
+      "body": ""
```

Comment:
We need some help updating the body text here :)

Reply:
Here is my suggestion!

Reply:
Great, thanks 🙏